dev-util/cmake/cmake-3.7.2.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A cross-platform build system"
+DESCRIPTION="CMake is a family of tools designed to build, test and package \
+software. It is used to control the software compilation process using \
+simple platform and compiler independent configuration files. CMake generates \
+native makefiles and workspaces that can be used in the compiler environment \
+of your choice."
+HOMEPAGE="https://cmake.org/"
+COPYRIGHT="2002-2016 Kitware, Inc., Insight Consortium"
+LICENSE="CMake"
+REVISION="1"
+SOURCE_URI="https://cmake.org/files/v3.7/cmake-$portVersion.tar.gz"
+CHECKSUM_SHA256="dc1246c4e6d168ea4d6e042cfba577c1acd65feea27e56f5ff37df920c30cae0"
+SOURCE_DIR="cmake-$portVersion"
+PATCHES="cmake-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 arm"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cmake$secondaryArchSuffix = $portVersion compat >= 3
+	cmd:cmake = $portVersion compat >= 3
+	cmd:ccmake = $portVersion compat >= 3
+	cmd:cpack = $portVersion compat >= 3
+	cmd:ctest = $portVersion compat >= 3
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libarchive$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libform$secondaryArchSuffix
+	#lib:jsoncpp$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	#lib:libuv$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libarchive$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libform$secondaryArchSuffix
+	#devel:jsoncpp$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	#devel:libuv$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku_devel
```

Comment:
I guess you should remove "haiku_devel" from BUILD_**PRE**REQUIRES because you already have
```
	haiku${secondaryArchSuffix}_devel
```
in BUILD_REQUIRES.